packages/vscode-extension/src/plugins/render-outlines/render-outlines-plugin.ts

```diff
@@ -22,80 +23,99 @@ const INSPECTOR_AVAILABILITY_MESSAGES = {
 export class RenderOutlinesPlugin implements ToolPlugin, RenderOutlinesInterface, Disposable {
   private eventEmitter = new EventEmitter();
   private isEnabled = false;
-  private devtoolsListeners: Disposable[] = [];
+  private eventListeners: Disposable[] = [];
```

Comment:
NIT: I'd rename this to follow the convention we use elsewhere
```suggestion
  private disposables: Disposable[] = [];
```

Reply:
Renamed in 9ef984f6